config/config.go

```diff
@@ -88,20 +89,6 @@ func (cfg Config) SummaryString() (summary string) {
 	return summary
 }
 
-// ReadFromFile reads a YAML configuration file from path
-func ReadFromFile(path string) (cfg *Config, err error) {
```

Comment:
I'm assuming this is removed to require users to use fill_struct, it'd be great to know how we _would_ read from a file using fill struct, even if just as a comment on this PR. Totally fine to see this deleted